config/locales/de.yml

```diff
@@ -1031,30 +1680,114 @@ de:
       format:
         delimiter: ''
   options:
+    admin: Administrator
+    aim: AIM
+    attending: attending
+    bus: Bus
     business: Geschäftlich
+    canceled: Canceled
     concert: Konzert
+    confirmed: confirmed
+    coordinator: Koordinator
+    crew: Besatzungsmitglied
+    declined: Abgelehnt
     dect: DECT
     djset: DJ-Set
     fax: Fax
     female: weiblich
     film: Film
+    icq: ICQ
+    idea: Idee
+    integrated: Integriert
+    jabber: Jabber / XMPP
     lecture: Vortrag
     lightning_talk: Lightning Talk
     male: männlich
     meeting: Meeting
     mobile: Mobil
+    msn: Msn
+    offer: Angebot
+    orga: Organisator
     other: Andere
+    otrs: OTRS
     performance: Theater/Performance
     phone: Telefon
     podium: Podiumsdiskussion
     private: Privat
-    secretary: Sekretärin/Vorzimmer
+    redmine: Redmine
+    reviewer: Reviewer
+    rt: Request Tracker
+    secretary: Sekretär/Vorzimmer
+    shuttle: Shuttle
     skype: Skype
+    speaker: Sprecher
+    submitter: Einreicher
     talk: Vortrag
+    unclear: Unklar
     work: Arbeit
     workshop: Workshop
+    yahoo: Yahoo
+  others_only: Nur 'others'
+  page_titles:
+    admin_users: frab - Benutzer mit Admininstrator-Rechten
+    change_password: Frab - Passwort ändern
+    crew_users: Benutzer mit Crew-Rechten
+    edit_account: Frab - Konto bearbeiten
+    login: Loggen Sie sich im frab ein
+    my_profile: Frab - Mein Profil
+    resend_confirmation: frab - Bestätigungsanweisungen erneut senden
+    signup: Erstellen Sie ein Konto im frab
+    unlock_instructions: frab - Anweisungen zum Entsperren des Kontos
+    welcome: Willkommen bei frab
   participate: Teilnehmen
   people: Personen
+  people_module:
+    add_person: Person hinzufügen
+    add_person_hint: Fügen Sie eine neue Person hinzu.
+    add_ticket: Ticket hinzufügen
+    add_ticket_hint: Erstellen Sie ein Ticket für diese Person.
+    all_people: Alle Personen
+    attending: Vor Ort
```

Comment:
maybe better "angekommen".
Then also in the next line "Markieren Sie die Person als angekommen"